### PR TITLE
Issue/13326 fix threats usecase

### PR DESCRIPTION
**Pull request by ashiagr**
Parent: #13795

This PR adds fix threats use case.

To test:
That `FixThreatsUseCaseTest` tests run fine and cover action results.

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/usecases/FixThreatsUseCaseTest.kt
Comment (ParaskP7) [inline WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/usecases/FixThreatsUseCaseTest.kt:58]: Minor (🔍): Instead of the `successfully` wording with the `when` part, I would have added a `given` part and preserve consistency with the above test, like: `given valid response, when fix threats is triggered, then Success is returned`

PS: When using `NetworkUnavailable`, `RemoteRequestFailure` and `Success` words on the test sentences, I always advocate in favor of normal works, so that everyone can read the test sentence without worrying about the implementation details. For example, I would have done something along these lines:
- `..., then the call fails due to network unavailability`
- `..., then the call fails`
- `..., then the call succeeds`

**Review by ParaskP7 — APPROVED** [review]
👋 @ashiagr !

I have reviewed and tested this PR, it all works as expected, kudos on the tests, they look so beautiful! ❤️ 

PS: I have left a comment just as an FYI, feel free to merge this PR.

File: WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/usecases/FixThreatsUseCaseTest.kt
Comment (ashiagr) [reply WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/usecases/FixThreatsUseCaseTest.kt:58]: > Instead of the successfully wording with the when part, I would have added a given part 

In 6c752ce

> When using NetworkUnavailable, RemoteRequestFailure and Success words on the test sentences, I always advocate in favor of normal works

I like it 👍, I'll start following this practice.

**Comment by ashiagr** [comment]
Thanks for the review @ParaskP7! I liked the FYI part and included it in this PR itself. Need approval before I merge it :).

File: WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/usecases/FixThreatsUseCaseTest.kt
Comment (ParaskP7) [reply WordPress/src/test/java/org/wordpress/android/ui/jetpack/scan/usecases/FixThreatsUseCaseTest.kt:58]: Looks great @ashiagr ! ❤️ 

I am now going to merge this PR.

**Pull request merged**
